.github/workflows/auto_assign_reviewers.yaml

```diff
@@ -0,0 +1,26 @@
+name: automatically assign reviewers
+
+on:
+  schedule:
+    - cron: "37 0 * * *" # At 00:37 UTC every day
+  push:
+
+jobs:
+  autoassign-reviewers:
+    name: assign automatically proposed reviewers
+    runs-on: ubuntu-latest
+    steps:
+      - name: download the list of proposed assignments
+        run: |
+          curl --output assignments.json https://leanprover-community.github.io/queueboard/automatic_assignments.json
+
+      - name: parse assignments file and assign reviewers
+        run: |
+          # We use a slightly roundabout way (shelling out to Python to parse the file and write
+          # the shell script we execute), as this is the easiest way to get around variable
+          # quotient issues.
+          # Rewriting this (or deciding that Python is nicer than shell and keeping it) is welcome.
+          python3 scripts/write_assignments.py
```

Comment:
I think this may be easier to maintain if we just do everything in the python script -- as is this I don't think this will work because the required secret won't be passed onto the scripts. I can give this a shot later this week.

Reply:
:+1: to moving the curl call into Python; I can do that later today